### PR TITLE
Feat/vadc workflow dashboard

### PR DESCRIPTION
**Pull request by jarvisraymond-uchicago**
Link to JIRA ticket if there is one: 
https://ctds-planx.atlassian.net/browse/VADC-1267

**Dev Notes**
Dashboard updates every 5 seconds using polling

Design: 
image

Implementation: 
In GWAS App: 
image

In Results App: 
image

In Storybook: 
output

### New Features
* Adds a dashboard to show the real time status of the user's workflow limit while using the Gen 3 GWAS App and Results App.

### CODE REVIEW

File: src/Analysis/SharedUtils/WorkflowLimitsDashboard/WorkflowLimitsUtils.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+  // validate data contains expected keys and they're numeric
+  if (
+    typeof data?.workflow_run !== 'number'
+    || typeof data?.workflow_limit !== 'number'
+  ) {
+    return false;
+  }
+  return true;
+};
```
Comment (pieterlukasse) [inline src/Analysis/SharedUtils/WorkflowLimitsDashboard/WorkflowLimitsUtils.js:31]: because of `workflowRun / workflowLimit` done in other part of the code, I think this function should also return `false` if `workflowLimit === 0`

**Review by pieterlukasse — COMMENTED** [review]
one small comment

File: src/Analysis/SharedUtils/WorkflowLimitsDashboard/WorkflowLimitsUtils.js (hunk 1 above)
Comment (jarvisraymond-uchicago) [reply src/Analysis/SharedUtils/WorkflowLimitsDashboard/WorkflowLimitsUtils.js]: Good catch, function updated along with unit tests for this edge case. 
```
  // validate data contains expected keys and they're numeric
  // and workflow limit is greater than 0
  if (
    typeof data?.workflow_run !== 'number'
    || typeof data?.workflow_limit !== 'number'
    || !(data.workflow_limit > 0)
  ) {
    return false;
  }
  ```

**Review by pieterlukasse — APPROVED** [review]
